config/initializers/riiif.rb

```diff
@@ -19,5 +19,5 @@
 
   # Riiif.not_found_image = 'app/assets/images/us_404.svg'
   #
-  Riiif::Engine.config.cache_duration = 365.days
+  Riiif::Engine.config.cache_duration = 30.days
```

Comment:
I think we may actually want the 365 day cache, from what I am reading this is client side caching.. so it would be beneficial for the client if they were to revisit without cost to server side. since our exhibit images are unlikely to change this shouldn't be an issue, its an instruction for the client's browser local cache